### DISCUSSION TITLE
Add keybind option for toggling animated emotes

### DISCUSSION THREAD

**Discussion by Vanidor** [Chatterino/chatterino2#4454] [Ideas]
Hi,

For people who have issues with photosensitivity, some emotes can be too flashy. The current solution to this problem is either to go into the settings and disable the emote animation or wait until the flashy emotes are no longer visible, and hide Chatterino in the background until then.

An easier solution would be to add the possibility of adding keybinds for toggling animated emotes.

**Comment by Felanbird** [comment]
While we aren't against adding this feature, it would require a bit more code than expected as we currently have no way of toggling a setting with the hotkey system.

**Reply by dnsge** [reply]
We actually do have a way of toggling settings with hotkeys already. See:
- `toggleLocalR9K`
- `setStreamerMode toggle`
- `setTabVisibility toggle`

For anyone wanting to implement this, relevant toggling code is in `src/widgets/Window.cpp` (do a search for "toggle") and in the corresponding hotkey actions in `src/controllers/hotkeys/ActionNames.hpp`.

**Reply by Felanbird** [reply]
well pajlada told me we didn't so blame him

**Comment by Mm2PL** [comment]
I'd rather actually have something automatic that limits rendering of animated emotes.
